core/src/main/java/org/veriblock/sdk/VeriBlockSecurity.java

```diff
@@ -85,17 +87,20 @@ private String bitcoinContextToString(List<BitcoinBlock> blocks) {
     }
 
     private String publicationToString(AltPublication publication) {
-        return String.format("ATV with VeriBlock tx id %s, endorsing to %s VeriBlock block, providing VeriBlock context (%s)",
+        return String.format("ATV with VeriBlock tx id %s, endorsing to %s:%d VeriBlock block, providing VeriBlock context (%s)",
                              publication.getTransaction().getId().toString(),
                              publication.getContainingBlock().getHash().toString(),
+                             publication.getContainingBlock().getHeight(),
                              veriblockContextToString(publication.getContext()));
     }
 
     private String publicationToString(VeriBlockPublication publication) {
-        return String.format("VTB with VeriBlock tx id %s in %s block, endorsing %s VeriBlock block to %s Bitcoin block, providing VeriBlock context (%s) and Bitcoin context (%s)",
+        return String.format("VTB with VeriBlock tx id %s in %s:%d block, endorsing %s:%d VeriBlock block to %s Bitcoin block, providing VeriBlock context (%s) and Bitcoin context (%s)",
```

Comment:
You could add some helper method to transform block to the "%s:%d" value.

Reply:
yeah just did this and some other minor improvements